packages/mjml-head-font/README.md

```diff
@@ -25,9 +28,11 @@ This tag allows you to import fonts if used in your MJML document
   </a>
 </p>
 
+<aside class="notice">
+  If you don't get a CSS file, model one on the "href" file in the example.
+</aside>
```

Comment:
Not needed with https://github.com/mjmlio/mjml/pull/2030/files#r486387185

Reply:
changed